_posts/2015-05-04-The-Role-of-the-API-Designer.md

```diff
@@ -0,0 +1,61 @@
+---
+title: The Role of the API Designer
+excerpt: The importance of an API Designer and the significant impact the role has on business today.
+layout: post
+date: 2015-05-04 09:00:00 +1000
+author: dave
+---
+
+APIs are everywhere, and enterprises want in on the action. Senior executives at various medium-to-large companies, like the one where I work, have been sold on the value that great APIs can bring - both for internal and external uses - and have started setting up Enterprise API Programs to support these efforts. But despite the best intentions and true commitment that got these programs funded and started, many obstacles remain to instilling an API-first culture that supports a great developer experience.
```

Comment:
Just wanted to make sure you're using the em dash here before "both" and after "uses."

— vs - (use the first)

When I copy and paste, it looks like you're using the hyphen. Em dash is Shift+Option+- on a mac.